### DISCUSSION TITLE
SP-API USER AMAZON SANDBOX

### DISCUSSION THREAD

**Discussion by giox851** [amzn/selling-partner-api-models#4350] [General] [answered]
Hi, I'm a developer and I need to do a simple test of retrieving orders from Amazon. Do I need the seller's user account or can I do it independently?

**Comment by chrisskhoury** [comment, marked as answer]
Hello @giox851,

You will need a seller user account to fetch the orders data on their behalf.

**Step 1**
- Create Seller Central account
- You must register for Seller Central account prior to registering as a developer.
- Only Professional Selling Accounts can register to develop or integrate with Selling Partner API. Individual accounts are not eligible. You can upgrade your account to a professional plan at any time.
- You must be a primary account user to complete registration.

**Step 2**
- Access developer console
- After creating a Seller Central account, select Apps and Services in the main menu, and select Develop Apps to access the Developer Console.

**Step 3**
- Complete developer profile
- The Developer Profile collects your organization’s contact information, information about the data you require in Amazon Services APIs, and security and use information to ensure your compliance with Amazon’s Acceptable Use Policy, Data Protection Policy, Developer Agreement, and access requirements.

After you complete the Developer Profile, Amazon will evaluate your information and create a case with next steps. You may be asked to provide additional information or documentation, so you must complete all fields truthfully and accurately.

Please refer to these resources to get started:

- Amazon Developer Website: https://developer.amazonservices.com/
- SP API Developer Documentation: https://developer-docs.amazon.com/sp-api/docs/sp-api-registration-overview


Thanks
Chris, Solution Architect
